packages/bruno-app/src/components/Sidebar/Collections/index.js

```
@@ -76,6 +78,50 @@ const Collections = ({ showSearch, isCreatingCollection, onCreateClick, onDismis
+          <div className="empty-collection-message">
+            <div className="indent-block" style={{ width: 16, minWidth: 16, height: '100%' }}>
+              &nbsp;
+            </div>
+            <div style={{ paddingLeft: 8 }}>
+              <button className="ml-1 add-request-link">+ Add request</button>
+            </div>
+          </div>
+        );
+      case 'empty-folder-message':
+        // we can calculate indents from item.item.depth
+        const indents = Array.from({ length: item.item.depth + 1 }).map((_, i) => i);
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)